tests/suites/test_suite_hkdf.function

```diff
@@ -14,12 +14,12 @@ void test_hkdf( int md_alg, char *hex_ikm_string, char *hex_salt_string,
 {
     int ret;
     size_t ikm_len, salt_len, info_len, okm_len;
-    unsigned char ikm[1024] = { '\0' };
-    unsigned char salt[1024] = { '\0' };
-    unsigned char info[1024] = { '\0' };
-    unsigned char expected_okm[1024] = { '\0' };
-    unsigned char okm[1024] = { '\0' };
-    unsigned char okm_string[1000] = { '\0' };
+    unsigned char ikm[128] = { '\0' };
+    unsigned char salt[128] = { '\0' };
+    unsigned char info[128] = { '\0' };
+    unsigned char expected_okm[256] = { '\0' };
+    unsigned char okm[256] = { '\0' };
```

Comment:
Why not using 128 Bytes here, too, if that's sufficient for our current tests?

Reply:
I chose arbitrarily the sizes before using a debugger to understand real sizes used. I enlarged the size of `okm` as it's the biggest value. I could reduce the size to 128. In fact we could reduce even further, but I think we should keep some extra size, for future possible tests.

Reply:
I tested with 128 bytes for the okm, and although the okm length is 82 bytes, the test
~~~
HKDF RFC5869 Test Vector #2
~~~
corrupts the stack. probably because of internal hkdf expansion, I haven't investigated it thorough.

Reply:
I would like to make sure we understand that failure before we go with 256 Bytes - I am happy to look at it at some point, or you do it, I'm fine either way.

Reply:
I looked at it some more.
The corruption happened in the size of `okm_string`. This is the stirng representation of `okm`, so it should be twice the size of `okm`, so, when it was smaller then 164 ( 82*2), the hexify function overflowed the buffer.
I changed `okm` and `expected_okm` to 128, and modified the `okm_string` to 256

Reply:
@RonEld Thanks for checking and explaining!